setup.py

```diff
@@ -51,7 +51,7 @@
 
     # Some of the tests use Conch:
     "PyCrypto==2.6.1",
-    "pyasn1==0.1.7",
+    "pyasn1==0.1.8",
```

Comment:
I think a safer change would be to pin the version of pyasn1_modules to one that's compatible with pyasn1.